src/model/CMakeLists.txt

```diff
@@ -1975,10 +1978,11 @@ set(${target_name}_test_src
   test/CoilHeatingWaterToAirHeatPumpVariableSpeedEquationFitSpeedData_GTest.cpp
   test/CoilHeatingWaterBaseboard_GTest.cpp
   test/CoilHeatingWaterBaseboardRadiant_GTest.cpp
+  test/CoilPerformanceDXCooling_GTest.cpp
```

Comment:
Alphabetize.

Reply:
its between S and W, no?

Reply:
Sorry, no action needed here. I was just noting what you had already done.